mailyak.go

```diff
@@ -194,3 +195,29 @@ func (m *MailYak) getFromAddr() string {
 func (m *MailYak) getAuth() smtp.Auth {
 	return m.auth
 }
+
+// stripNames returns a new slice with only the email parts from the RFC 5322 addresses.
+//
+// Or in other words, converts:
+// ["a@example.com", "John <b@example.com>", "invalid"]
+// to
+// ["a@example.com", "b@example.com", "invalid"].
+//
+// Note that invalid addresses are kept as they are.
+func stripNames(addresses []string) []string {
+	result := make([]string, 0, len(addresses))
+
+	for _, original := range addresses {
+		addr, err := mail.ParseAddress(original)
```

Comment:
It looks like `ParseAddress()` has some [known quirks](https://github.com/golang/go/issues/38447) but by using the original input if an parse error occurs, this should be fine / backwards compatible :+1: